tests/src/HTMLElement/elements/HTMLDocumentElement.js

```diff
@@ -0,0 +1,29 @@
+/*global test*/
+import assert from 'proclaim';
+import {readFileSync} from 'fs';
+
+const expect = assert.strictEqual;
+const throws = assert.throws;
+
+const lib = '../../../../lib/';
+
+const Document = require(lib + 'Document');
+const Doctype = require(lib + 'Doctype');
+
+const fileText = readFileSync(__dirname + '/../../../data/linkstest.html', { encoding: 'utf-8' });
```

Comment:
And:
- `document.documentElement.innerHTML = '<body>test</body>';`
of course :)